imagersite/imager_images/views.py

```diff
@@ -1,3 +1,140 @@
-from django.shortcuts import render
+from __future__ import unicode_literals
+from __future__ import division
+from django.views.generic import DetailView
+from django.views.generic.edit import CreateView, UpdateView
+from imager_images.models import Photo, Album, Face
+from django.db.models import Q
+from django.http import Http404
+import os
 
-# Create your views here.
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+def get_faces(photo):
+    import Algorithmia
+    import base64
+    Algorithmia.apiKey = os.environ['ALGORITHMIA_KEY']
+    file_path = 'media/' + str(photo.file)
+    file_path = os.path.join(BASE_DIR, file_path)
```

Comment:
Shouldn't you be able to get this from the MEDIA_ROOT setting?  Eventually you might not actually want to have the media root in the same directory as your executable code.  What then?